viewer/app/index.html

```diff
@@ -16,7 +16,7 @@
   <link rel="stylesheet" href="styles/bundled.css">
   <link rel="canonical" href="https://googlechrome.github.io/lighthouse/viewer/">
 </head>
-<body class="lh-vars">
```

Comment:
Do we need this? `lh-vars` is added by to the root element by the report renderer.